wizeline/wize-q-remix
# refactor: add airbnb eslint

#### What does this PR do?
- Change ESLint rules
- Apply the new rules to the project. 

#### Where should the reviewer start?
- Run the project and verify that you don't get any error. 
- Run the test and verify that all passed. `npm run test:integration`

#### How should this be manually tested?
1. Login
2. Verify you can create all the following scenarios: 
       -  create question as user
       - create a question anonymously 
       - comment a question as user 
       - comment anonymously 
       - edit a comment
       - delete a comment
       - upvote and downvote a comment 
       - vote for a question
       - apply different filters. 
       - disable a question. 
       [admins]
       - answer a question
       - edit an answer
       - delete an answer
       - approve a comment as an answer.
       - Unapproved a comment as an answer.
3. :tada:


#### What are the relevant tickets?
Closes #50 

#### Screenshots
<img width="556" alt="Screen Shot 2022-12-09 at 9 54 22" src="https://user-images.githubusercontent.com/97203617/206741295-600c7894-83eb-47d3-bc39-e51ba7745d07.png">

https://user-images.githubusercontent.com/97203617/206740617-fb8707b0-1442-4f65-be8c-01614b3a2653.mp4
